src/Areas/Marketplace/Commands/Product/ProductGetCommand.cs

```diff
@@ -0,0 +1,187 @@
+    private readonly Option<string> _languageOption = OptionDefinitions.Marketplace.Language;
+    private readonly Option<string> _marketOption = OptionDefinitions.Marketplace.Market;
+    private readonly Option<bool> _lookupOfferInTenantLevelOption = OptionDefinitions.Marketplace.LookupOfferInTenantLevel;
+    private readonly Option<bool> _includeHiddenPlansOption = OptionDefinitions.Marketplace.IncludeHiddenPlans;
+    private readonly Option<string> _planIdOption = OptionDefinitions.Marketplace.PlanId;
+    private readonly Option<string> _skuIdOption = OptionDefinitions.Marketplace.SkuId;
+    private readonly Option<bool> _includeServiceInstructionTemplatesOption = OptionDefinitions.Marketplace.IncludeServiceInstructionTemplates;
+    private readonly Option<string> _partnerTenantIdOption = OptionDefinitions.Marketplace.PartnerTenantId;
+    private readonly Option<string> _pricingAudienceOption = OptionDefinitions.Marketplace.PricingAudience;
+    private readonly Option<string> _objectIdOption = OptionDefinitions.Marketplace.ObjectId;
+    private readonly Option<string> _altSecIdOption = OptionDefinitions.Marketplace.AltSecId;
+    public override string Name => "get";
+
+    public override string Description =>
+        $"""
+        Retrieves a single private product (offer) for a given subscription from Azure Marketplace.
+        Returns detailed information about the specified marketplace product including plans, pricing, and metadata.
+        
+        Required options:
+        - --{OptionDefinitions.Common.SubscriptionName}: Azure subscription ID
+        - --{OptionDefinitions.Marketplace.ProductIdName}: Product ID to retrieve
+        
+        Optional filtering options:
+        - --{OptionDefinitions.Marketplace.LanguageName}: Product language (default: en)
+        - --{OptionDefinitions.Marketplace.MarketName}: Product market (default: US)
+        - --{OptionDefinitions.Marketplace.PlanIdName}: Filter by specific plan ID
+        - --{OptionDefinitions.Marketplace.SkuIdName}: Filter by specific SKU ID
+        
+        Optional inclusion options:
+        - --{OptionDefinitions.Marketplace.IncludeStopSoldPlansName}: Include stop-sold plans
+        - --{OptionDefinitions.Marketplace.IncludeHiddenPlansName}: Include hidden plans
+        - --{OptionDefinitions.Marketplace.IncludeServiceInstructionTemplatesName}: Include service instruction templates
+        - --{OptionDefinitions.Marketplace.LookupOfferInTenantLevelName}: Check tenant private audience
+        
+        Optional header options:
+        - --{OptionDefinitions.Marketplace.PartnerTenantIdName}: Partner tenant ID
+        - --{OptionDefinitions.Marketplace.PricingAudienceName}: Pricing audience
+        - --{OptionDefinitions.Marketplace.ObjectIdName}: AAD user ID
+        - --{OptionDefinitions.Marketplace.AltSecIdName}: Alternate Security ID (for MSA users)
+        """;
```

Comment:
Descriptions don't need to describe the parameters.  We'll likely be moving away from this as the tool list command has all the parameter info.  I would remove it for now.